### DISCUSSION TITLE
Searx self-hosting questions

### DISCUSSION THREAD

**Discussion by anonymouschad** [searx/searx#3215] [General]
Hello, I'm new to searx and want to know more about how to self-hosting privately. As far as I'm understanding, connecting your server to other search engines will reveal your IP to them and I don't want to use a VPS either since you're just moving your trust from an instance to that VPS. Can anyone suggest a way that can help me to hide my IP address in a private instance? If there is no way then can you tell me the most private public instance? Finally how much computer resources and bandwidth will searx take up for two people to use?

**Comment by kvch** [comment]
You can find more info here about private instances: https://searx.github.io/searx/user/own-instance.html

In general, I do not think you should worry about your IP. Google, Facebook, Youtube, etc. do not care about your IP. They track you based on cookies in your browser. Think about it. If they tracked you based on IP you would get the same recommendations as the people who connect to the Internet using the same IP as you.

If you really want to hide your IP, use [Tor](https://www.torproject.org/).
